### PR TITLE
3.2 updates

### PR DESCRIPTION
**Pull request by itamarhaber**

### CODE REVIEW

File: commands/client-reply.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+* `ON`. This is the default mode in which the server returns a reply to every command.
+* `OFF`. In this mode the server will not reply to client commands.
+* `SKIP`. This mode skips the reply of command immediately after it.
+
+@return
+
+When called with either `OFF` or `SKIP` subcommands, no reply is made. When called with `ON`:
+
+@simple-string-reply: `OK` if the connection name was successfully set.
```
Comment (badboy) [inline commands/client-reply.md]: This sentence doesn't match the command.

File: commands/client-reply.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Sometimes it can be useful for clients to completely disable replies from the Redis server. For example when the client sends fire and forget commands or performs a mass loading of data, or in caching contexts where new data is streamed constantly. In such contexts to use server time and bandwidth in order to send back replies to clients, which are going to be ignored, is a shame.
```
Comment (badboy) [inline commands/client-reply.md]: "is a shame" sounds weird in this descriptive context of a command

File: commands/client-reply.md (hunk 1 above)
Comment (itamarhaber) [inline commands/client-reply.md]: Resolved.

File: commands/client-reply.md (hunk 2 above)
Comment (itamarhaber) [inline commands/client-reply.md]: Resolved.

**Comment by badboy** [comment]
Last merge for the year 2015.
Next: Party!

**Comment by itamarhaber** [comment]
201<s>5</s>6

**Pull request merged**